app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -153,6 +153,11 @@ class _QuickJumpState extends ConsumerState<QuickJump> {
     ];
   }
 
+  void routeTo(String name) {
+    if (context.canPop()) context.pop();
+    context.pushNamed(name);
+  }
+
```

Comment:
can we do this typed, please?

```suggestion
  void routeTo(Route route) {
    if (context.canPop()) context.pop();
    context.pushNamed(route.name);
  }

```